.github/workflows/build-verify.yml

```
@@ -58,44 +61,40 @@ jobs:
           severity: 'CRITICAL,HIGH'
-          # Use the .trivyignore file to suppress specific vulnerability alerts.
-          trivyignores: .trivyignore 
\ No newline at end of file
+          # Use a .trivyignore file to manage a persistent list of ignored CVEs.
+          # This provides a clear, version-controlled record of accepted risks.
+          ignore-file: '.trivyignore'
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v3
+        with:
+          sarif_file: 'trivy-results.sarif' 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)